docs/md/README.md

```diff
@@ -287,18 +287,21 @@ There are some unit tests under `./test/zero/`.
 
 Run them with `boot test`.
 
-There are also some integration tests under `./integration`.
-Run them using aliases defined in the `./deps.edn` file.
+There are some integration tests under `integration`. These are not yet
+automated so we suggest running them locally before submitting.
+To run the integration tests, you'll need a `wfl` server running locally:
+
+```shell
+./ops/server.sh
+```
+
+Then run the tests via the aliases defined in `deps.edn` file.
```

Comment:
Maybe describe the syntax for selecting individual tests in Kaocha.

Reply:
I've updated the `Integration Testing` section in `development process` to capture this. I've also included a link to their documentation.